source/reference/compatibility.txt

```diff
@@ -97,8 +98,11 @@ To specify a read preference, you can use either:
 
 - The :method:`Mongo.setReadPref()` method.
 
-The legacy ``mongo`` shell required running ``rs.secondaryOk()`` to
-enable secondary reads.
+- When using ``mongosh`` to connect directly to a :term:`secondary` 
```

Comment:
[suggestion]

Since the bulleted list is meant to be a list of ways to specify a read preference, we might be better off bringing this out of the bulleted list and just having it be its own paragraph.

Reply:
Removed the - per suggestion.